core/src/main/resources/error/error-classes.json

```diff
@@ -2272,6 +2282,16 @@
     ],
     "sqlState" : "42P01"
   },
+  "WINDOW_FUNCTION_AND_FRAME_MISMATCH" : {
+    "message" : [
+      "<wf> function can only be evaluated in an ordered row-based window frame with a single offset: <w>."
```

Comment:
Let's assign proper names of parameters here and other places.

Reply:
How about `WINDOW_SPEC_MISSING_ORDER_BY_OR_WITHOUT_SINGLE_OFFSET` ?
or `INVALID_WINDOW_SPEC_FOR_FRAME_LESS_OFFSET_WINDOW_FUNCTION`.